lib/KCard/index.vue

```diff
@@ -350,6 +351,9 @@
         }
         return {};
       },
+      isThumbnailPresent() {
```

Comment:
Will need to be removed in favor of checking `thumbnailDisplay` configuration. https://github.com/learningequality/kolibri-design-system/pull/752#pullrequestreview-2265059042